crates/krator/Cargo.toml

```diff
@@ -50,8 +50,8 @@ tracing = { version = "0.1", features = ['log'] }
 tracing-futures = "0.2"
 
 [dev-dependencies]
-kube-derive = "0.48"
-schemars = "0.8.0"
+kube-derive = "0.51"
+schemars = "0.8.2"
```

Comment:
Nit: Should this just be `0.8`?

Reply:
Ah, yeah I guess so, I just wanted to be sure it updated because it patched a clippy lint.